src/mdx-components.tsx

```diff
@@ -0,0 +1,18 @@
+import type { MDXComponents } from 'mdx/types';
+
+/**
+ * Let's not mess with the MDX components
+ *
+ * @param components
+ * @returns
+ */
```

Comment:
I like the straightforwardness of this docstring.

Reply:
Yeah LOL